tests/level2_test.go

```diff
@@ -0,0 +1,84 @@
+package caf_tests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSharedServicesResourceGroupsExists(t *testing.T) {
+	t.Parallel()
+
+	test := prepareTestTable()
+
+	client, _ := azure.GetResourceGroupClientE(test.SubscriptionID)
+
+	var top int32 = 100
+
+	result, _ := client.List(context.Background(), "tagName eq 'level' and tagValue eq 'level2'", &top)
+
+	rgList := result.Values()
+
+	expected := 4
```

Comment:
See above. expected should be 3 for this configuration.

Reply:
updated test, thanks for the feedback